x-pack/plugins/security_solution/public/cases/components/all_cases/table_filters.test.tsx

```diff
@@ -151,4 +151,20 @@ describe('CasesTableFilters ', () => {
     );
     expect(onFilterChanged).toHaveBeenCalledWith({ reporters: [{ username: 'casetester' }] });
   });
+
+  it('StatusFilterWrapper should have a fixed width', () => {
+    const wrapper = mount(
+      <TestProviders>
+        <CasesTableFilters {...props} />
+      </TestProviders>
+    );
+
+    expect(wrapper.find('[data-test-subj="status-filter-wrapper"]').first()).toHaveStyleRule(
```

Comment:
More of a personal preference, but I think the test is more resilient if you just trigger a resize event and make sure the bounding box of the element doesn't change. That way, this test doesn't fail if UX decided to change the actual size by a few pixels.

Reply:
@michaelolo24 Interesting! I would like to enforce that the minimum width is `180px`. Can I test that with the resize?

Reply:
Oh this way is better then if you want to explicitly confirm the 180px size. I just figured you didn't want the size of the wrapper to change. Maybe change the description to be more specific like `it('StatusFilterWrapper should have a fixed width of 180px')` to make your intent more explicit